lib/verk/workers_manager.ex

```diff
@@ -90,8 +90,13 @@ defmodule Verk.WorkersManager do
 
   @doc false
   def handle_info(:enqueue_inprogress, state) do
-    :ok = QueueManager.enqueue_inprogress(state.queue_manager_name)
-    {:noreply, state, 0}
+    case QueueManager.enqueue_inprogress(state.queue_manager_name) do
```

Comment:
Ah I see now what you meant in https://github.com/edgurgel/verk/pull/102/files#r92948579 👍